### ISSUE TITLE
Github action warning

### ISSUE THREAD

**Issue by themyth92** [getsentry/action-release#14]
Hi, when I use this action on Production, there is a warning from this module.
The warning from Github is:
```
Unexpected input(s) 'version', valid inputs are ['entryPoint', 'args', 'environment', 'sourcemaps', 'finalize', 'started_at']
```

I believe it is because of the `version` input from the yml file.
My yml file is
```
# master.yml
...
- uses: getsentry/action-release@v1.0.0
  with:
      environment: production
      sourcemaps: './dist'
      version: 1.1.1
```

The release recorded from Sentry is correct and reflect the correct version from the yml file so It is not a big issue.

**Comment by emrebuyukozkan** [comment]
Experiencing the same problem with the latest verion.

**Comment by thomasmichaelwallace** [comment]
Me three- looks like the release is based on an older [action.yml](https://github.com/getsentry/action-release/blob/5d9568fbc36d8c781d40c78e25f9926df978e4ea/action.yml) that doesn't include `version`.

I'm going to fork and do a release that matches this for my own benefit. [link pending]

I'll also contact support [case pending]

**Comment by mgaeta** [comment]
Hi all.
I omitted `version` from action.yml in `v1.0.0` and then fixed it in `v1.0.1`. I just updated the documentation and pushed new tags so you should be able to fix your code by switching to `v1`:
```
- uses: getsentry/action-release@v1
```
